sdk-setup/src/mb2

```diff
@@ -1363,7 +1363,8 @@ try_to_make_spec() { # Tries to create a missing spec
     [[ -f "$1" ]] && return # It's not missing
     local yaml="${1%.spec}.yaml"
     [[ -f "$yaml" ]] || return # No yaml
-    ANSI_COLORS_DISABLED=1 specify -n -N "$yaml" || fatal "Failed to convert .yaml to .spec"
+    ANSI_COLORS_DISABLED=1 specify -n -N "$yaml" \
+        || fatal "Failed to convert YAML to SPEC file"
```

Comment:
Messages are without context, so maybe saying RPM with SPEC would be better. Applies to all messages.